### ISSUE TITLE
git diff option differ between version 1.2.0 and 1.3.0

### ISSUE THREAD

**Issue by polosskevin** [scalameta/scalafmt#1084]
- **Version**: 1.3.0
- **Integration**: CI
- **Configuration**: not dependant

## Steps

I ran scalafmt like this:

```bash
scalafmt --test --git true --diff
```

## Problem

Scalafmt output:
1.2.0:
All files are formatted with scalafmt :)
1.3.0:
```
Exception in thread "main" org.scalafmt.Error$NoMatchingFiles$: No files formatted/tested. Verify your configured include/exclude filters and supplied command line arguments.
        at org.scalafmt.Error$NoMatchingFiles$.<clinit>(Error.scala)
        at org.scalafmt.cli.Cli$.run(Cli.scala:157)
        at org.scalafmt.cli.Cli$.mainWithOptions(Cli.scala:66)
        at org.scalafmt.cli.Cli$.main(Cli.scala:61)
        at org.scalafmt.cli.Cli.main(Cli.scala)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at coursier.Bootstrap.main(Bootstrap.java:428)
```
## Expectation

I would like the formatted output to look like this:
```scala
All files are formatted with scalafmt :)
```

## Workaround
downgrade to 1.2.0
&
## Notes

I was trying to add scalafmt on some continuous integration of one of my project. May be i'm not using scalafmt correctly, but i would personnally expect the 1.2.0 output in my case. As i can easily modify some non scala file in the project and still expect the tools not to return an error.

**Comment by olafurpg** [comment]
Thank you for reporting! Looks like this was fixed in #1093

**Comment by olafurpg** [comment]
Fix has been released as 1.4.0
